runtime/autoload/netrw.vim

```diff
@@ -77,6 +77,14 @@ fun! netrw#ErrorMsg(level,msg,errnum)
    return
   endif
 
+  if g:netrw_use_notify == 1
```

Comment:
there is already a `g:netrw_use_errorwindow` option below... why is this new option needed?

Reply:
I did consider adding it to the existing variable, but refrained from doing so for discoverability. I often find these multi-value variables pretty difficult to reason about, because you have to search the help text, rather than the variable entry to find out if it supports the behavior you want. `g:netrw_use_errorwindow` already has three different values, that do pretty much the same thing, and adding this to it would also overload it further, possibly adding to the confusion.

That being said, if you would rather have it as an extra option on `g:netrw_use_errorwindow` there's nothing stopping us from doing that.. I also agree that the default behavior is pretty much unusable, so I wouldn't be against setting this as the new default, but I don't know how people generally react to "breaking" changes?